curriculum/challenges/english/10-coding-interview-prep/data-structures/create-a-priority-queue-class.english.md

```diff
@@ -33,10 +33,14 @@ tests:
     testString: assert((function(){var test = new PriorityQueue();  return (typeof test.dequeue === 'function')}()));
   - text: Your <code>PriorityQueue</code> class should have a <code>size</code> method.
     testString: assert((function(){var test = new PriorityQueue();  return (typeof test.size === 'function')}()));
+  - text: Your <code>PriorityQueue</code> class should have a <code>front</code> method.
+    testString: assert((function(){var test = new PriorityQueue();  return (typeof test.front === 'function')}()));
   - text: Your <code>PriorityQueue</code> class should have an <code>isEmpty</code> method.
     testString: assert((function(){var test = new PriorityQueue();  return (typeof test.isEmpty === 'function')}()));
   - text: Your <code>PriorityQueue</code> class should correctly keep track of the current number of items using the <code>size</code> method as items are enqueued and dequeued.
     testString: assert((function(){var test = new PriorityQueue(); test.enqueue(['David Brown', 2]); test.enqueue(['Jon Snow', 1]); var size1 = test.size(); test.dequeue(); var size2 = test.size(); test.enqueue(['A', 3]); test.enqueue(['B', 3]); test.enqueue(['C', 3]); return (size1 === 2 && size2 === 1 && test.size() === 4)}()));
+  - text: The <code>front</code> method should return the correct item at the front of the queue as items are enqueued and dequeued.
+    testString: assert((function(){var test = new PriorityQueue(); test.enqueue(["David Brown", 2]); var front1 = text.front(); test.enqueue(["Jon Snow", 1]); var front2 = test.front(); test.dequeue(); test.enqueue(["A", 3]); var front3 = test.front(); test.enqueue(["B", 3]); test.enqueue(["C", 3]); test.dequeue(); var front4 = test.front(); return (front1 === "David Brown" && front2 === "Jon Snow" && front3 === "David Brown" && front4 === "A");})());
```

Comment:
There seems to be a typo in the `testString`.  It should be `var front1 = test.front();` instead of `var front1 = text.front();`
```suggestion
    testString: assert((function(){var test = new PriorityQueue(); test.enqueue(["David Brown", 2]); var front1 = test.front(); test.enqueue(["Jon Snow", 1]); var front2 = test.front(); test.dequeue(); test.enqueue(["A", 3]); var front3 = test.front(); test.enqueue(["B", 3]); test.enqueue(["C", 3]); test.dequeue(); var front4 = test.front(); return (front1 === "David Brown" && front2 === "Jon Snow" && front3 === "David Brown" && front4 === "A");})());
```